lib/coins.py

```diff
@@ -43,6 +43,7 @@
 from server.block_processor import BlockProcessor
 import server.daemon as daemon
 from server.session import ElectrumX, DashElectrumX
+from functools import partial
```

Comment:
Python stdlib imports go in the first block of imports, as per generally accepted coding style